jenkins-scripts/docker/gz-abichecker.bash

```diff
@@ -51,29 +51,29 @@ else
   export ABI_JOB_HEADER_PREFIX=${ABI_JOB_SOFTWARE_NAME/[ignz]*-/}[0-9]*
 fi
 
-# check if NEED_C17_COMPILER should be set
+# check if NEED_C17_COMPILER should be set. Only needed for Bionic so only care about Citadel
```

Comment:
Now that we have dropped Bionic, the whole `NEED_C17_COMPILER` logic can be removed right?

Reply:
We still have support for Citadel until the end of year and it can only run on Ubuntu Bionic. Let's keep it by now. We can remove the whole support when Citadel is finished.